### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by yura-enttsel**
https://yura-enttsel.github.io/react_movies-list-filter-js/

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
We have a problem with demo link. Please check it
image

**Comment by yura-enttsel** [comment]
> We have a problem with demo link. Please check it image

And also there is a small problem with creating PR. Wrong base repository is chosen by default. 
image

File: src/App.jsx [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
+    ? moviesList.filter(({ title, description }) => (
+      title.toLowerCase().includes(queryToSearch)
+        || description.toLowerCase().includes(queryToSearch)
+    ))
+    : moviesList;
+}
+
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const saveValueToQuery = value => setQuery(value);
```
Comment (volodymyr-soltys97) [inline src/App.jsx:19]: Consider renaming 'saveValueToQuery' to 'updateQuery' for better readability and to convey the purpose of the function more clearly.

**Review by volodymyr-soltys97 — APPROVED** [review]
Nice work 👍

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**